src/Model/CustomerView/FilterDefinition.php

```diff
@@ -48,10 +48,10 @@ class FilterDefinition extends AbstractModel
     /** @var bool */
     private $shortcutAvailable = false;
 
-    /** @var int|null */
+    /** @var string|null */
```

Comment:
why string? column type is `bigint(20)`

Reply:
@dvesh3 Is set to string here https://github.com/pimcore/customer-data-framework/blob/59753f6dd7e1b98f40c9d4350cfbbc792d557d49/src/Model/CustomerView/FilterDefinition/Dao.php#L100-L104 and the column type is `DATETIME` see https://github.com/pimcore/customer-data-framework/blob/59753f6dd7e1b98f40c9d4350cfbbc792d557d49/src/Resources/sql/filterDefinition/datamodel.sql#L9-L10